src/renderer/corporate/member/corporate-add-member-container.jsx

```diff
@@ -128,22 +216,35 @@ class CorporateAddMemberContainer extends PureComponent {
 }
 
 const mapStateToProps = (state, props) => {
-	const parentIdentity = identitySelectors.selectCorporateProfile(state, {
-		identityId: props.match.params.parentId
+	const identity = identitySelectors.selectIdentity(state);
+	let { parentId } = props.match.params;
+	parentId = parentId || identity.id;
+
+	const parentProfile = identitySelectors.selectCorporateProfile(state, {
+		identityId: parentId
 	});
 
+	if (!parentProfile || parentProfile.identity.type !== 'corporate') {
+		throw new Error(`Invalid parent identity, requires 'corporate' type`);
+	}
+
 	return {
-		parentIdentity,
-		basicAttributeTypes: identitySelectors.selectBasicCorporateAttributeTypes(state),
-		basicIdentity: identitySelectors.selectCorporateProfile(state, {
-			identityId: props.match.params.identityId
-		}),
+		parentId,
+		parentProfile,
+		identity,
```

Comment:
shouldn't identity always point to `parentProfile.identity`
If so, `selectIdentity` should not be called, instead `selectCorporateProfile` should be called without props (orr with `{ identityId: null }`. It will fetch the current identity inside it's `identity` field

Reply:
I want to make the component flexible enough to be opened by different identity.
But it doesn't make sense, removed on https://github.com/SelfKeyFoundation/Identity-Wallet/pull/1832/commits/ab4f1f8df3920fec8b77f8adfb05399c32f2a40f